docs/configuration/profile.md

```diff
@@ -32,6 +32,56 @@ Add a packer build
 * `source_ami` The source AMI ID for an `amazon-ebs`
 * `ssh_username` Default `ubuntu`
 * `ami_virtualization_type` Default `hvm`
+* `tagging_role` the name of an IAM role that exists in each remote account that allows the AMI to be retagged
+
+  Example usage:
+
+  <pre>
```

Comment:
You can use [fenced code blocks](https://help.github.com/articles/creating-and-highlighting-code-blocks/) instead of `<pre>` tags to get syntax highlighting. But don't let changing that block shipping this.